src/assets.rs

```diff
@@ -269,33 +271,52 @@ impl HighlightingAssets {
             .map(|syntax| SyntaxReferenceInSet { syntax, syntax_set }))
     }
 
-    fn find_syntax_by_extension(&self, e: Option<&OsStr>) -> Result<Option<SyntaxReferenceInSet>> {
+    fn find_syntax_by_extension(
```

Comment:
Is it not possible to leave this function unchanged and only strip suffixes in the two places we stripped suffixes before this PR?

Reply:
You are right, we don't need to change this.